molecule/fedora/molecule.yml

```diff
@@ -28,7 +39,7 @@ provisioner:
       force_valid_group_names: always
   inventory:
     host_vars:
-      debian9:
+      debian9_systemd:
```

Comment:
Is this needed in an fedora only testing configuration?

Reply:
I will remove it wholesale from the skeleton once we can drop Debian 9. Until then I opted to keep it around just to keep `molecule.yml` files for the more specific scenarios in sync with those for the `default` scenario.